apps/myjobs/config/puma.rb

```diff
@@ -0,0 +1,56 @@
+# Puma can serve each request in a thread from an internal thread pool.
```

Comment:
We don't use puma. We could safely delete this file.

Reply:
done.